src/osmviz/manager.py

```diff
@@ -387,16 +387,8 @@ def createOSMImage(self, bounds, zoom):
         )
 
 
-class _useragenthack(FancyURLopener):
-    def __init__(self, *args):
-        FancyURLopener.__init__(self, *args)
-        for i, (header, val) in enumerate(self.addheaders):
-            if header == "User-Agent":
-                del self.addheaders[i]
-                break
-        self.addheader("User-Agent", "OSMViz/1.1.0 +https://hugovk.github.io/osmviz")
-
-
 # import httplib
 # httplib.HTTPConnection.debuglevel = 1
-urllib.request._urlopener = _useragenthack()
+opener = urllib.request.build_opener()
+opener.addheaders = [('User-agent', 'OSMViz/1.1.0 +https://hugovk.github.io/osmviz')]
```

Comment:
Next lint nit, Black formatting:

```suggestion
opener.addheaders = [("User-agent", "OSMViz/1.1.0 +https://hugovk.github.io/osmviz")]
```